src/screens/title.rs

```diff
@@ -22,7 +22,7 @@ enum TitleAction {
     Exit,
 }
 
-fn enter_title(mut commands: Commands) {
+fn show_title_menu(mut commands: Commands) {
```

Comment:
```suggestion
fn show_title_screen(mut commands: Commands) {
```

Or just `show_title`, depending on the choice for consistency.

Reply:
Disagree. There may be multiple kinds of title screen. Depending on your view, the splash may also be part of the title. This is specifically the title screen *menu*

Reply:
The splash isn't part of the title screen though, because we explicitly have `Screen::Splash` distinct from `Screen::Title`. I've also never heard the term "title menu", only "title screen" and "main menu"